src/components/TripItem.jsx

```diff
@@ -59,9 +60,13 @@ export const TripItem = ({ geojson, trip, hasDateHeader }) => {
   const day = useMemo(() => f(getStartDate(trip), 'dddd DD MMMM'), [f, trip])
 
   const CO2 = useMemo(() => {
+    if (mode) {
+      const CO2TripByMode = computeCO2TripByMode(trip, mode)
+      return Math.round(CO2TripByMode * 100) / 100
```

Comment:
nit: the rounded mechanism is used twice. Maybe we can extract it inside a utils.

by the way, I wonder if the "compute" methods should be in charge of this 'rounding' feature. I think so